python/kiss_icp/kiss_icp.py

```diff
@@ -86,11 +93,14 @@ def get_adaptive_threshold(self):
             else self.adaptive_threshold.get_threshold()
         )
 
-    def get_prediction_model(self, frame_delta):
+    def get_prediction_model(self, frame_delta_ratio):
         if len(self.poses) < 2:
             return np.eye(4)
         model = np.linalg.inv(self.poses[-2]) @ self.poses[-1]
-        return np.linalg.matrix_power(model, int(frame_delta))
+        if frame_delta_ratio == 1:
+            return model
+        else:
+            return np.linalg.matrix_power(model, int(round(frame_delta_ratio)))
```

Comment:
linalg matrix_power takes time.

Without this change, the processing time increases 15%